src/main/java/dev/jbang/cli/Export.java

```diff
@@ -502,3 +498,239 @@ private Path getJlinkOutputPath() {
+}
+
+@Command(name = "gradle", description = "Exports a Gradle project")
+class ExportGradleProject extends BaseExportProject {
+
+	@Override
+	String getType() {
+		return "gradle";
+	}
+
+	@Override
+	void renderBuildFile(BuildContext ctx, Path projectDir, String fullClassName) throws IOException {
+		Project prj = ctx.getProject();
+		ResourceRef templateRef = ResourceRef.forResource("classpath:/export-build.qute.gradle");
+		Path destination = projectDir.resolve("build.gradle");
+
+		List<MavenRepo> repositories = prj.getRepositories();
+		List<String> dependencies = prj.getMainSourceSet().getDependencies();
+		// Turn any URL dependencies into regular GAV coordinates
+		List<String> depIds = dependencies	.stream()
+											.map(JitPackUtil::ensureGAV)
+											.collect(Collectors.toList());
+		// And if we encountered URLs let's make sure the JitPack repo is available
+		if (!depIds.equals(dependencies)
+				&& repositories.stream().noneMatch(r -> DependencyUtil.REPO_JITPACK.equals(r.getUrl()))) {
+			prj.addRepository(DependencyUtil.toMavenRepo(DependencyUtil.ALIAS_JITPACK));
+		}
+
+		TemplateEngine engine = TemplateEngine.instance();
+		Template template = engine.getTemplate(templateRef);
+		if (template == null)
+			throw new ExitException(EXIT_INVALID_INPUT, "Could not locate template named: '" + templateRef + "'");
+		String result = template
+								.data("group", group)
+								.data("artifact", artifact)
+								.data("version", version)
+								.data("description", prj.getDescription().orElse(""))
+								.data("repositories", repositories	.stream()
+																	.map(MavenRepo::getUrl)
+																	.filter(s -> !"".equals(s))
+																	.collect(Collectors.toList()))
+								.data("javaVersion", prj.getJavaVersion())
+								.data("gradledependencies", gradleify(depIds))
+								.data("fullClassName", fullClassName)
+								.render();
+		Util.writeString(destination, result);
+	}
+
+	private List<String> gradleify(List<String> collectDependencies) {
+		return collectDependencies.stream().map(item -> {
+			if (item.endsWith("@pom")) {
+				return "implementation platform ('" + item.substring(0, item.lastIndexOf("@pom")) + "')";
+			} else {
+				return "implementation '" + item + "'";
+			}
+		}).collect(Collectors.toList());
+	}
+}
+
+@Command(name = "maven", description = "Exports a Maven project")
+class ExportMavenProject extends BaseExportProject {
+
+	@Override
+	String getType() {
+		return "maven";
+	}
+
+	@Override
+	void renderBuildFile(BuildContext ctx, Path projectDir, String fullClassName) throws IOException {
+		Project prj = ctx.getProject();
+		ResourceRef templateRef = ResourceRef.forResource("classpath:/export-pom.qute.xml");
+		Path destination = projectDir.resolve("pom.xml");
+
+		Map<String, String> properties = new HashMap<>();
+		if (prj.getJavaVersion() != null) {
+			properties.put("maven.compiler.source", prj.getJavaVersion());
```

Comment:
this is not handling case of `//JAVA 17+` it shuold remove the +.

Reply:
Thanks for pointing it out. I'll modify it to use `JavaUtil.javaVersion` which should handle `+` cases.

Reply:
Done